gremlin-go/driver/graph.go

```diff
@@ -97,14 +98,15 @@ func (p *Path) String() string {
 	return fmt.Sprintf("path[%s]", strings.Trim(strings.Join(strings.Fields(fmt.Sprint(p.objects)), ", "), "[]"))
 }
 
-func (p *Path) GetPathObject(key string) (error, interface{}) {
+// GetPathObject returns the value that corresponds to the key for the Path.
```

Comment:
minor godoc nit? : `returns the value that corresponds to the key for the Path and error if the value is not present or cannot be retrieved`